### ISSUE TITLE
Better Representation of Catalog columns

### ISSUE THREAD

**Issue by rainwoodman** [bccp/nbodykit#375]
Several users expected 'print' to provide a preview of the content of the `dask.array`.

I am not sure how we can do this...

**Comment by nickhand** [comment]
We could theoretically use a default chunk size internally, with a very small header chunk, that could be printed out without too much cost?

**Comment by rainwoodman** [comment]
I was to add this monkey patch in `__init__.py`..

```

# see https://github.com/dask/dask/issues/2628
def patch_dask_array_str():
    import dask.array as da
    import numpy

    oldstr = da.Array.__str__

    def dastr(self):
        s = oldstr(self)𝄾
        try:
            if len(s) > 0:
                s += ' # first: %s' % str((self[0].compute()))
            if len(s) > 1:
                s += ' # last: %s' % str((self[-1].compute()))
        finally:
            return s

    da.Array.__str__ = dastr

patch_dask_array_str()
```

**Comment by rainwoodman** [comment]
But combinded with the fact that we want to use the compute method of the source instead of dask.array, we probably want to introduce a subclass of it.

**Comment by rainwoodman** [comment]
Closed by #381
